Documentation/install/upgrade.rst

```diff
@@ -331,6 +331,11 @@ Deprecated options
   or later to 1.4.x may result in disruption of connections that connect via
   services.
 
+* ``lb``: The ``--lb`` feature has been deprecated. It has not been in use and
+  was well tested. If you need load-balancing with pickup from a device
```

Comment:
Nits:
1. s/was well tested/has not been well tested/
2. s/If you need load-balancing with pickup from a device directly/If you need load-balancing on a particular device/